src/element.ts

```diff
@@ -77,17 +77,6 @@ export abstract class Element {
     return `auto${Element.ID++}`;
   }
 
-  /**
-   * Default font for text. This is not related to music engraving. Instead, see `Flow.setMusicFont(...fontNames)`
-   * to customize the font for musical symbols placed on the score.
-   */
-  static TEXT_FONT: Required<FontInfo> = {
-    family: Font.SANS_SERIF,
-    size: Font.SIZE,
-    weight: FontWeight.NORMAL,
-    style: FontStyle.NORMAL,
-  };
-
```

Comment:
In VexFlow 4.x, we had fonts for styling "text-like" elements, and fonts for styling "music symbols". `TEXT_FONT` was used for the "text-like" elements.... i.e., anything that could accept CSS style font declarations (e.g., '15px Arial, sans-serif;`).

By deleting TEXT_FONT, are you saying that going forward, all elements, whether they are "text-like" or "music symbols" will all go through the same path for font styling? They will all use CSS-style font declarations?

Reply:
Thanks. This is helpful.